data/avro-schema/src/main/resources/avro/net/corda/data/membership/command/registration/mgm/VerifyMember.avsc

```diff
@@ -4,5 +4,23 @@
   "namespace": "net.corda.data.membership.command.registration.mgm",
   "doc": "Command issued when a member registration request has been validated and the MGM must communication with the member for verify information provided.",
   "fields": [
+    {
```

Comment:
Seeing how this is used in the runtime-os, I think it would be better to remove this change and add the mgm to the registration state which is passed with events.

Reply:
done